spec/menu_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'menu'
+
+describe Menu do 
+
+  it 'should have a list of available dishes' do
+   expect(subject).to respond_to(:dishes_available)
+  end
+
+  it 'should have available dishes' do
+    expect(subject.dishes_available).to be_truthy
```

Comment:
I'd argue that all this is testing, is that `dishes_available` exists, not that there's anything useful in it...